tests/bluetooth/tester/src/btp_bap.c

```diff
@@ -525,6 +539,22 @@ static void stream_started(struct bt_bap_stream *stream)
 	struct audio_stream *a_stream = CONTAINER_OF(stream, struct audio_stream, stream);
 
 	LOG_DBG("Started stream %p", stream);
+
+	a_stream->seq_num = 0;
+
+	/* Run audio clock work in system work queue */
+	k_work_init_delayable(&a_stream->audio_clock_work, audio_clock_timeout);
+	k_work_schedule(&a_stream->audio_clock_work, K_USEC(0));
+
+	/* Run audio send work in user defined work queue */
+	k_work_queue_init(&iso_data_work_q);
+	k_work_queue_start(&iso_data_work_q, iso_data_thread_stack_area,
+			   K_THREAD_STACK_SIZEOF(iso_data_thread_stack_area),
+			   ISO_DATA_THREAD_PRIORITY, NULL);
+	k_work_init_delayable(&a_stream->audio_send_work, audio_send_timeout);
+	k_work_schedule_for_queue(&iso_data_work_q, &a_stream->audio_send_work,
+				  K_USEC(a_stream->stream.qos->interval));
```

Comment:
One important part here is that the seq_num starts at 0 when the CIS is connected, and since you schedule the `audio_send_work` here with a delay, it should probably be incremented before first TX. 

Alternatively, use `K_NO_WAIT` when starting the first TX here

Reply:
@mkasenberg is there a reason to not start sending immediately?

Reply:
The seq_num is incremented in `audio_clock_work` and TX is perforemed by `audio_send_work`. But just in case, let's start seq_num from 1:
```c
a_stream->seq_num = 1;
```

Reply:
Sorry, did not refreshed the site. I was not sure

> @mkasenberg is there a reason to not start sending immediately?

I was not sure if I can rely on priority of system work queue.

Reply:
> I was not sure if I can rely on priority of system work queue.

That is a fair point :)